roles/dnsmasq/tasks/main.yml

```diff
@@ -68,6 +68,12 @@
     resolvconffile: >-
       {%- if resolvconf.rc == 0 -%}/etc/resolvconf/resolv.conf.d/head{%- else -%}/etc/resolv.conf{%- endif -%}
 
+- name: Cleanup existing resolv.conf file
```

Comment:
note: this might only purge things when there is no resolvconf package installed, and the target file is not the "head", but the /etc/resolv.conf instead